src/accessors/IEnvironmentRead.ts

```diff
@@ -0,0 +1,10 @@
+import { ISettingRead } from './ISettingRead';
+/**
+ * Allows read-access to configuration and other data which is not created by the normal user
+ */
+
+export interface IEnvironmentRead{
+    settings(): ISettingRead;
+
+    environmentVariables(): object; //todo
```

Comment:
With the environment variables, are you thinking the rocketlets should have access to them all or only a limited subset? As I'm thinking only the limited subset